### PR TITLE
Journey to nyx booster packs

### PR DESCRIPTION
**Pull request by Peter-Kang**
Init JOU changes

### CODE REVIEW

**Comment by Peter-Kang** [comment]
Also fixed card vs name array, and added the ratio

File: server/routes/booster-generation/sets/jou.js [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+            //put the land in 
+            booster.push(lands[getRandomIndex(lands.length)]);
+            //Unpackages the card
+            boosters.push(booster.map(mapCard));
+        }
+    }
+    return boosters;
+}
+
+exports.generatePacks = generatePacks;
```
Comment (cman131) [inline server/routes/booster-generation/sets/jou.js:256]: missing an empty new line at the end of the file. ES6 style standard ;p

Comment (cman131) [reply server/routes/booster-generation/sets/jou.js:256]: Another style comment is that this seems to use 4 spaces for indentation. This project uses 2.

Comment (cman131) [reply server/routes/booster-generation/sets/jou.js:256]: These aren't essential and I can merge this and then fix the style. Just good to know for future PRs

**Review by cman131 — APPROVED** [review]

**Pull request merged**